pkg/util/testing/wrappers.go

```diff
@@ -331,3 +333,286 @@ func (e *EventRecordWrapper) Message(message string) *EventRecordWrapper {
 func (e *EventRecordWrapper) Obj() EventRecord {
 	return e.EventRecord
 }
+
+// ResourceClaimSpecBuilder provides a common interface for building ResourceClaimSpec
+type ResourceClaimSpecBuilder struct {
+	spec resourcev1.ResourceClaimSpec
+}
```

Comment:
Hm this is ok on its own, but for consistency with the rest of the codebase I would propose calling Wrapper, wdyt?